package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "synapse-react-client",
-  "version": "1.15.27",
+  "version": "1.15.28",
```

Comment:
I just merged some icon changes and updated the version to 1.15.28. Not sure if this is going to cause conflict here.